notebooks/growth_model_eda.ipynb

```diff
@@ -0,0 +1,768 @@
+{
```

Comment:
<p>Awsm! Thanks Hema.. </p><p>I guess we can now merge it to main branch. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>